src/components/AddOffering/TabPanels.js

```diff
@@ -0,0 +1,24 @@
+import { Grid } from "@material-ui/core";
```

Comment:
This is currently not used. May use in future when I implement the ability to "add postings as offerings"